apps/contracts/scripts/mint-lions.ts

```diff
@@ -35,77 +60,36 @@ function getImage(path: string) {
 }
 
 function processLionData(data: LionData) {
-  const {
-    gender,
-    birthday,
-    coalition,
-    prides,
-    unique_features,
-    whisker_right_0,
-    whisker_right_1,
-    whisker_right_2,
-    whisker_left_0,
-    whisker_left_1,
-    whisker_left_2,
-    ear_left,
-    ear_right,
-    face,
-    mouth,
-    id,
-    name,
-  } = data;
-
-  const image = getImage(`${face}`);
-
-  const attributes = [
-    { trait_type: 'id', value: id },
-    { trait_type: 'gender', value: gender },
-    { trait_type: 'birthday', value: birthday },
-    { trait_type: 'coalition', value: coalition },
-    {
-      trait_type: 'prides',
```

Comment:
also best to be prides_controlled, not to be confunded with pride it belongs to.

Reply:
ok, will change

Reply:
@cerealkill Right now `prides` field has prides names and `prides_controlled` has number of prides that controlled by the lion. So, we should rename both fields, any thoughts?

<img width="606" alt="image" src="https://user-images.githubusercontent.com/271810/217761932-ad1a04ef-5ac0-4c11-996f-05cacc66db09.png">